packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/StartUp/PluginHandler.cs

```diff
@@ -30,6 +31,7 @@ internal void SelectPlugin(CommandLineOptions options)
                 {
                     CodeModelPlugin.Instance = plugin.Value;
                     CodeModelPlugin.Instance.IsNewProject = options.IsNewProject;
+                    CodeModelPlugin.Instance.Emitter = emitter;
```

Comment:
Can we just construct the emitter here?

Reply:
Program needs the emitter to write debug information when we attach the debugger.